github/ipfs.yml

```diff
@@ -6741,137 +6595,9 @@ teams:
         - stefanhans
     privacy: closed
   IPFS Camp 2019:
-    members:
-      member:
-        - 1015bit
-        - abn
-        - achingbrain
-        - aeddi
-        - aknuds1
-        - akru
-        - alanshaw
-        - alekswn
-        - andrewxhill
-        - angiemaguire
-        - anorth
-        - apronkin
-        - AquiGorka
-        - arsstone
-        - aschmahmann
-        - AuHau
-        - autonome
-        - avras
-        - b5
-        - balupton
-        - bigs
-        - bjoyce3
-        - blackforestboi
-        - carsonfarmer
-        - cesarosum
-        - cmk
-        - codynhat
-        - cpacia
-        - cwaring
-        - cyborgshead
-        - daijiale
-        - dankelleher
-        - darkdh
-        - daviddias
-        - dborzov
-        - deltazxm
-        - dignifiedquire
-        - doctorrobinson
-        - dongtianyi
-        - drbh
-        - DrCatman
-        - ec1oud
-        - emerick
-        - enricomarino
-        - eshon
-        - gfanton
-        - glouvigny
-        - gnunicorn
-        - gorhgorh
-        - Gozala
-        - gpestana
-        - gregdhill
-        - guillemcordoba
-        - hannahhoward
-        - hareeshnagaraj
-        - hinshun
-        - hsanjuan
-        - hugomrdias
-        - ianamunoz
-        - ianopolous
-        - jacobheun
-        - jamiejn
-        - janjanovna
-        - jbenet
-        - jdelgadopin
-        - jimpick
-        - jkarni
-        - joaosantos15
-        - jonnycrunch
-        - JustMaier
-        - kanej
-        - kinu
-        - kk3wong
-        - koalalorenzo
-        - Kubuxu
-        - leshokunin
-        - lidel
-        - listenaddress
-        - ma82
-        - mafintosh
-        - magnshen
-        - MAkcanca
-        - maparent
-        - MarneeDear
-        - MichaelMure
-        - moeghashim
-        - momack2
-        - moul
-        - neogeweb3
-        - obo20
-        - olizilla
-        - parkan
-        - pepoospina
-        - pooja
-        - Prtfw
-        - ratoshniuk
-        - raulk
-        - requilence
-        - rittme
-        - rklaehn
-        - roignpar
-        - romaric-juniet
-        - Schwartz10
-        - ShishKabab
-        - smihaylov
-        - Stebalien
-        - stefanhans
-        - steven004
-        - teh-f00l
-        - terichadbourne
-        - vasa-develop
-        - vojtechsimetka
-        - waynewyang
-        - whyrusleeping
-        - Wondertan
-        - yrliou
-        - yunigraham
-        - zeckli
     privacy: closed
   IPFS HTTP Client Maintainers:
```

Comment:
@galargh should we remove teams which end up with no members?
There is a bunch of them below (`Repos - Python`, `Repos - Rust`, `Repos - Scala` etc).
It does not have to be as part of this cleanup, ideally would be separate PR, and also follow `# KEEP: ` convention.

Reply:
Yes, let's remove the empty teams but I'd prefer doing it after this round of user changes is complete if you don't mind.